### ISSUE TITLE
FC when I enter encryption details

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#180]
```
D/WifiService( 2232): ACTION_SCREEN_ON
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_SCREEN_OFF
D/WifiService( 2232): setting ACTION_DEVICE_IDLE timer for 900000ms
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): got ACTION_DEVICE_IDLE
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=1 state=8]
V/WifiStateTracker( 2232): Changing supplicant state: COMPLETED ==> DORMANT
D/WifiStateTracker( 2232): Deconfiguring interface and stopping DHCP
I/WifiMonitor( 2232): Event [CTRL-EVENT-DISCONNECTED - Disconnect event - 
remove keys]
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=-1 state=8]
I/WifiMonitor( 2232): Event [CTRL-EVENT-DRIVER-STATE STOPPED]
V/WifiStateTracker( 2232): New network state is DISCONNECTED
V/WifiStateTracker( 2232): Changing supplicant state: DORMANT ==> DORMANT
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
D/WifiService( 2232): ACTION_SCREEN_ON
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
I/WifiMonitor( 2232): Event [CTRL-EVENT-DRIVER-STATE STARTED]
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=-1 state=2]
V/WifiStateTracker( 2232): Changing supplicant state: DORMANT ==> SCANNING
I/WifiMonitor( 2232): Event [ SCAN-RESULTS ]
I/WifiMonitor( 2232): Event [ SCAN-RESULTS ]
I/WifiMonitor( 2232): Event [WPS-AP-AVAILABLE ]
I/WifiMonitor( 2232): Event [Trying to associate with 00:18:f6:75:4e:9f 
(SSID='BTHomeHub-12F2' freq=2412 MHz)]
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=-1 state=3]
V/WifiStateTracker( 2232): Changing supplicant state: SCANNING ==> ASSOCIATING
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=1 state=4]
V/WifiStateTracker( 2232): Changing supplicant state: ASSOCIATING ==> ASSOCIATED
I/WifiMonitor( 2232): Event [Associated with 00:18:f6:75:4e:9f]
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=1 state=7]
V/WifiStateTracker( 2232): Changing supplicant state: ASSOCIATED ==> COMPLETED
I/WifiMonitor( 2232): Event [CTRL-EVENT-CONNECTED - Connection to 
00:18:f6:75:4e:9f completed (reauth) [id=1 id_str=]]
V/WifiStateTracker( 2232): New network state is CONNECTED
V/WifiStateTracker( 2232): mWakeLock.acquired
D/WifiStateTracker( 2232): DhcpHandler: DHCP request started
V/WifiStateTracker( 2232): DhcpHandler: DHCP request succeeded
V/WifiStateTracker( 2232): mWakeLock.released
V/WifiStateTracker( 2232): IP configuration: ipaddr 192.168.1.78 gateway 
192.168.1.254 netmask 255.255.255.0 dns1 192.168.1.254 dns2 0.0.0.0 DHCP server 
192.168.1.254 lease 86400 seconds
D/WifiWatchdogService( 2232): (android.server.ServerThread) BTHomeHub-12F2 
(00:18:f6:75:4e:9f) does not require the watchdog
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_SCREEN_OFF
D/WifiService( 2232): setting ACTION_DEVICE_IDLE timer for 900000ms
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_SCREEN_ON
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_SCREEN_OFF
D/WifiService( 2232): setting ACTION_DEVICE_IDLE timer for 900000ms
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): ACTION_BATTERY_CHANGED pluggedType: 2
I/WifiService( 2232): handleMessage: MESSAGE_ENABLE_WIFI
I/WifiService( 2232): handleMessage: MESSAGE_START_WIFI
D/WifiService( 2232): got ACTION_DEVICE_IDLE
I/WifiService( 2232): handleMessage: MESSAGE_STOP_WIFI
I/WifiMonitor( 2232): Event [CTRL-EVENT-STATE-CHANGE id=1 state=8]
V/WifiStateTracker( 2232): Changing supplicant state: COMPLETED ==> DORMANT
D/WifiStateTracker( 2232): Deconfiguring interface and stop
```

Original issue reported on code.google.com by `tormod.m...@gmail.com` on 27 Sep 2010 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
Are you using the latest version, 0.2.8? There was a bug that I fixed and 
re-released right after releasing 0.2.7.
```

Original comment by `rdamazio@gmail.com` on 27 Sep 2010 at 5:22
- Changed state: **Waiting**

**Comment by GoogleCodeExporter** [comment]
```
Got no replies, closing.
```

Original comment by `rdamazio@gmail.com` on 31 Oct 2010 at 5:22
- Changed state: **Invalid**
